packages/env/lib/init-config.js

```diff
@@ -153,10 +157,16 @@ function dockerFileContents( image, config ) {
 		}
 	}
 
+	// @todo: test compat.
+	if ( config.xhprof !== 'off' ) {
```

Comment:
If the xhprof flag is a boolean (see above), this could be written like `config.xhprof === true`